src/Driver.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace PodPoint\MonologKinesis;
+
+use Illuminate\Contracts\Container\Container;
+use Illuminate\Support\Arr;
+use Monolog\Logger;
+use PodPoint\MonologKinesis\Contracts\Client;
+
+class Driver
```

Comment:
I was initially a bit confused by this class, could it be better named? It looks a bit like a 'Kinesis logger factory'

Alternatively we could potentially inline this into the service provider - there doesn't look that much to it and that's the only place it's used

Reply:
Yeah I kind of agree. I think everything was in the Service Provider to start with but I wanted to try to organise things a bit better. I'll move it back there and see how we feel.